app/views/profiles/show.html.erb

```diff
@@ -9,14 +9,27 @@
     <%= image_tag 'linha.png', class: 'linha' %>
     <p>Associate other accounts with your profile. This way, you can use any of
     them to authenticate in the Call4Paperz:</p>
-    <br />
+    <br>
     <ul>
       <% @profile.unassociated_providers.each do |provider| %>
         <li><%= link_to(provider.show_name, "auth/#{provider.name}") %></li>
       <% end %>
     </ul>
     <% end %>
-  </div>
+
+    <% unless @profile.authentications.empty? %>
```

Comment:
`if @profile.authentications.any?` is not a better option here?
Also, can you put a print here?